aten/src/ATen/native/sparse/SparseTensor.cpp

```diff
@@ -244,11 +244,11 @@ Tensor sparse_to_dense(const SparseTensor& self) {
   return dst.add_(self);
 }
 
-SparseTensor& copy_sparse_(SparseTensor& self, const SparseTensor& src) {
+SparseTensor& copy_sparse_(SparseTensor& self, const SparseTensor& src, bool non_blocking) {
```

Comment:
Hmm does this work if `src` and `self` are on different cuda devices?

Reply:
Yes, this works:
```
i = torch.LongTensor([[0, 2], [1, 0], [1, 2]]).t()
v = torch.FloatTensor([3, 4, 5])
size = torch.Size([2,3])
x = torch.sparse_coo_tensor(i, torch.ones_like(v), size, device=torch.device('cuda:0'))
x2 = torch.sparse_coo_tensor(i, v, size, requires_grad=True, device=torch.device('cuda:1'))
x.copy_(x2)
#####################
torch.cuda.sparse.FloatTensor of size (2,3) with indices:
tensor([[0, 1, 1],
        [2, 0, 2]], device='cuda:0', grad_fn=<Error>)
and values:
tensor([3., 4., 5.], device='cuda:0', grad_fn=<Error>)
```